markdown/chapters/appendices/resources.md

```diff
@@ -68,8 +70,21 @@ Assume English unless specified otherwise.
 Videos from non-regular streamers or Doomsday content creators.  
 Assume English unless specified otherwise.
 
+### 2022
+
+ - Awaiting Content :D
+
```

Comment:
Probably should wait for content to add the heading imo

Reply:
Removed